monkestation/code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -255,9 +255,14 @@
 	// and if we fail to revive them, don't proceede the chain
 	sac_target.adjustOxyLoss(-100, FALSE)
 	sac_target.grab_ghost() // monke edit: try to grab their ghost
+
 	if(!sac_target.heal_and_revive(50, span_danger("[sac_target]'s heart begins to beat with an unholy force as they return from death!")))
 		return
 
+	//monkestation addition start:
+	sac_target.reagents.remove_all() //stops chems from killing in the mansus
```

Comment:
```suggestion
	sac_target.reagents?.remove_all() //stops chems from killing in the mansus
```
i don't trust this cursed game to not have weird edge-cases where reagents is somehow null